runner/runner.go

```diff
@@ -0,0 +1,366 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"sync/atomic"
+	"syscall"
+	"time"
+)
+
+type siteInfo struct {
+	Multisite int
+	Siteurl   string
+	Disabled  int
+}
+
+type site struct {
+	URL string
+}
+
+type event struct {
+	URL       string
+	Timestamp int
+	Action    string
+	Instance  string
+}
+
+var (
+	wpCliPath string
+	wpNetwork int
+	wpPath    string
+
+	workersGetEvents int
+	workersRunEvents int
+
+	heartbeatInt int
+
+	disabledLoopCount    uint64
+	eventRunErrCount     uint64
+	eventRunSuccessCount uint64
+
+	logger  *log.Logger
+	logDest string
+	debug   bool
+)
+
+const getEventsLoop time.Duration = time.Minute
+const getEventsBreak time.Duration = time.Second
+const runEventsBreak time.Duration = time.Second * 10
+
+func init() {
+	flag.StringVar(&wpCliPath, "cli", "/usr/local/bin/wp", "Path to WP-CLI binary")
+	flag.IntVar(&wpNetwork, "network", 0, "WordPress network ID, `0` to disable")
+	flag.StringVar(&wpPath, "wp", "/var/www/html", "Path to WordPress installation")
+	flag.IntVar(&workersGetEvents, "workers-get", 1, "Number of workers to retrieve events")
+	flag.IntVar(&workersRunEvents, "workers-run", 5, "Number of workers to run events")
+	flag.IntVar(&heartbeatInt, "heartbeat", 60, "Heartbeat interval in seconds")
+	flag.StringVar(&logDest, "log", "os.Stdout", "Log path, omit to log to Stdout")
```

Comment:
Since this is intended to be run as a daemon controlled through Sysvinit/systemd, I wanted consistency in how the runner's options are set. That said, the answer to your question is yes, the defaults file could handle log destinations.